spec/rails_helper.rb

```diff
@@ -24,7 +24,7 @@
 
 Capybara.register_driver :headless_chrome do |app|
   capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(
-    chromeOptions: { args: %w(headless disable-gpu) }
+    chromeOptions: { args: %w[headless disable-gpu] }
```

Comment:
Not sure why Rubocop complains here. Completely valid.

I would still move Capybara related stuff to its own `spec/support/...` file.

Reply:
Yeah, I thought that was a weird one!

Do I simply `require: "selenium-webkit"` config in the spec/rails_helper.rb?

Reply:
No need, because any file in `spec/support` is required on line 8